src/99-appendices/08-coordinate-systems.md

```diff
@@ -116,47 +116,35 @@ below, the origin is at the AC and the orientation of the axes is RAS. Unless
 
-Hybrid spaces combining surface and volume data used with CIFTI files.
+### Nonstandard template identifiers
 
-| Coordinate System | Description                                                                                                                                                                                                                                                                                                                                                                                                           |
-| ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| HCPMNIfsLR32k     | Combination of `MNI152NLin6AsymConte69` subcortical volumetric space (with 2mm grid) and `fsLR32k` surface space resulting in 91282 data points per timepoint. Used in Human Connectom Project. [https://github.com/Washington-University/HCPpipelines/tree/master/global/templates/91282_Greyordinates](https://github.com/Washington-University/HCPpipelines/tree/master/global/templates/91282_Greyordinates)      |
-| HCPMNIfsLR59k     | Combination of `MNI152NLin6AsymConte69` subcortical volumetric space (with 1.6mm grid) and `fsLR59k` surface space resulting in 170494 data points per timepoint. Used in Human Connectom Project. [https://github.com/Washington-University/HCPpipelines/tree/master/global/templates/170494_Greyordinates](https://github.com/Washington-University/HCPpipelines/tree/master/global/templates/170494_Greyordinates) |
+| Coordinate System | Description                                                                                                                                                                                                                                                           |
+| ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| individual        | Participant specific anatomical space (for example derived from T1w and/or T2w images). This coordinate system requires specifying an additional, participant-specific file to be fully defined. In context of surfaces this space has been refered to as `fsnative`. |
```

Comment:
just clarifying that the individual anatomical space can come from any MR/PET modality. 

also, i know we are doing fsnative, but how would we handle things like CIVET or other algorithms that generate surface spaces. is `fsnative` the default key for native space, independent of the algorithm used to produce it?

Reply:
It looks like fsnative is not in the current standard, so it is reasonable to me to change it, if people want. What about `recon` (short for `individualsurface`)?

Speaking of `individual`, is this a proposal, or was there previous discussion? Were there other options? It just feels long...

Reply:
yeah I also feel like "individual" is too long..but do we have a good alternative?

Reply:
> @satra:
> how would we handle things like CIVET or other algorithms that generate surface spaces

what you mean by "surface space"? how those operate differently w.r.t. FreeSurfer and fsnative/fsaverage?

Reply:
yes, they are different surfaces similar to fs32LR being different from fsnative/fsaverage

Reply:
I am probably being obtuse here, my apologies. But fsLR is a template space, for CIVET we would then add the corresponding ID for their template (for instance).

How that maps on to the individual space?

Reply:
i'm not a user, so it would be difficult for me to say (perhaps some of the folks north of the border can chime in). in terms of common templates, i would definitely say the freesurfer templates are much more widely used.